week3_HM/todo/src/components/Task.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
```

Comment:
It looks like you're no longer using this component in week 3? If so, please remove it from your codebase.